frappe/public/js/frappe/desk.js

```diff
@@ -214,19 +214,20 @@ frappe.Application = class Application {
 
 	email_password_prompt(email_account,user,i) {
 		var me = this;
+		const email_id = email_account[i]["email_id"];
 		let d = new frappe.ui.Dialog({
 			title: __('Password missing in Email Account'),
 			fields: [
 				{
 					'fieldname': 'password',
 					'fieldtype': 'Password',
-					'label': __('Please enter the password for: <b>{0}</b>', [email_account[i]["email_id"]]),
+					'label': __('Please enter the password for: <b>{0}</b>', [email_id], "Email Account"),
 					'reqd': 1
 				},
 				{
 					"fieldname": "submit",
 					"fieldtype": "Button",
-					"label": __("Submit")
+					"label": __("Submit", null, "Submit password for Email Account")
```

Comment:
```suggestion
					"label": __("Submit", context="Submit password for Email Account")
```

Reply:
Let's use keyword argument instead of passing null
Let's do this for other places too

Reply:
Hey @hasnain2808, JavaScrip doesn't have named parameters and this is [in accordance with the docs](https://frappeframework.com/docs/v13/user/en/translations#6-adding-context-for-a-string).

Reply:
@barredterra thanks for pointing that out, mixed up python here.